testsuite/classlibrary/TestSynthDescMetadata.sc

```diff
@@ -0,0 +1,57 @@
+TestSynthDescMetadata : UnitTest {
+	var dir, path, mdPath, metadata, original, oldMdPlugin;
+	var defname = \test_md;
```

Comment:
i think you can make this a `const`? or are symbol literals not compatible with const...

Reply:
Both done.